### DISCUSSION TITLE
Remove default title for newly uploaded images / Убрать заголовок по умолчанию для вновь загруженных фото

### DISCUSSION THREAD

**Discussion by kabalin** [PastVu/pastvu#450] [Ideas / Идеи и предложения]
Обсуждение: https://pastvu.com/news/141?hl=comment-2552073

Зачастую вновь загруженные снимки отправляются на модерацию с названием по умолчанию (название файла). Предполагается, что неопытный пользователь не догадывается его изменить из-за того, что поле уже заполнено. Как вариант предлагаю не устанавливать название файла в заголовок загруженного фото, в этом случае при первом редактировании поле будет иметь вид:

![image](https://user-images.githubusercontent.com/329780/152228469-eef443e3-70ec-499d-80e3-a96ccbf58f48.png)

В галереи пользователя вновь загруженные снимки будут иметь вид:

![image](https://user-images.githubusercontent.com/329780/152228678-37136f2b-ef22-4b73-a2f0-78515a90588f.png)

**Comment by anvaruk** [comment]
мне кажется имелось ввиду чтоб валидация поля не допускала пустого значения при оправке на публикацию

**Reply by kabalin** [reply]
Валидацию на поле можно тоже добавить. Вопрос скорее в том, если мы уберем оттуда названия файлов, не усложнит ли это процесс заполнения данных - скажем загрузил пользователь 10 снимков за раз, а они все "Без названия"? Насколько присутствие названия файла может быть для кого-то важно?

**Reply by mkgrgis** [reply]
Названия файлов из неоторых источников уже готовы для подписывания изображений. Не то чтобы такое бывает часто, но и не редко. По логике проще требовать, чтобы пользователь заводил курсор редактирвоания хотя бы один раз в поле названия, а без этого не пускать на публикацию и давать бледно-красный фон (и/или мигацющий?) поля названия.

**Reply by Bhowered** [reply]
Только не мигающий.
Всё вроде просто реализовано в других проектах.
Пишется строка "Поля со звёздочкой требуют обязательного заполнения"
Без заполнения таких полей, после нажатия кнопки "Отправить" появляется окно с требованием "Заполните требуемые поля".

**Reply by mkgrgis** [reply]
Можно писать в названии "Из файла " и имя файла. Запрещать отправку на публикацию пока пользователь не уберёт "Из файла" в начале строки. Слова "Из файла" можно писать красным и сделать подсказку о требованиях.

**Reply by paul-k-pastvu** [reply]
Валидация там есть - с пустым названием на модерацию не отправишь.
Но если  при загрузке вообще терять именя файлов - то это тоже не здорово.
Во-первых, есть люди, которые на это ориентируются, и имена файлов помогают им в работе.
Во-вторых, многие готовят файлы к загрузке, и заранее делают удобные именя, чтобы потом на сайте вносить минимум исправлений.

Последнее предложение мне нравится.
Иого, к.м.к., разумный путть:
* добавлять шаблонный текст (и всплывающую подсказку о том, что следует сделать)
* подсвечивать неизмененные поля названий
* не давать отправлять на модерацию без исправления
